src/client/styles/cards.css

```diff
@@ -157,6 +157,8 @@
 }
 
 .scaled-big {
+    margin-top: 155%;
+    position: absolute;
```

Comment:
This breaks the layout for the played card. For a large number of players it sits on the pass button
![Screenshot from 2021-07-30 10-21-38](https://user-images.githubusercontent.com/10673021/127625942-b25326dc-87b9-457a-afac-62f55700fa74.png)